pack/td.json

```diff
@@ -526,6 +747,75 @@
         "uniqueness": false
     },
     {
+        "advancement_cost": 4,
+        "agenda_points": 2,
+        "code": "13042",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "illustrator": "Ed Mattinian",
+        "keywords": "Security",
+        "pack_code": "td",
+        "position": 42,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Place 1 agenda counter on Armored Servers when you score it.\n<strong>Hosted agenda counter</strong>: For the remainder of this run, the runner must trash a card from his or her grip as an additional cost to jack out or break a subroutine. Use this ability only during a run.",
```

Comment:
FFG's templating is horrible and inconsistent :/

Here **_Runner_** is capitalized.